### ISSUE TITLE
the code about nerf's feature part for generating mask

### ISSUE THREAD

**Issue by hrz2000** [StarsTesla/RePaint-NeRF#2]
Thank you very much for your great work, I am very interested in training the feature part of the nerf, can you share it？, it will be a bit difficult for me to merge the DFF code myself.
My email is 19374275@buaa.edu.cn

**Comment by StarsTesla** [comment]
@hrz2000  Do you met any problems when you follow the insrtuction of DFF official code?

**Comment by hrz2000** [comment]
Thank you for your prompt response. I realized that the two libraries can run independently, if only need the consistency mask of the edited object. I am still trying the DFF library, and it seems to be going smoothly so far.

Additionally, I would like to ask you a question: if the mask got from SAMs is so precise, then if I want to generate a new object larger than this object or an incompatible new shape, the optimization process will be limited by this mask. If the unmask weight is set smaller, the background will change. Do you have any good solutions to address this issue

**Comment by StarsTesla** [comment]
Yes, there three things that would influence the results, the first one is the prompt obviously, the second will be the weight of the mask and unmask as you have noticed, and the third one is the learning rate, if you make it smaller, it will be much slower, but keep stable, larger will make the training faster but the whole scene will be influenced, maybe it will converge to a better effect, I think it might be the maske region(edited regieon) have optimized to be the most it could be followed the text prompt, then the unmask region should also tend to the reduce the loss. Lastly, the best way is to test thoes hyperparameters, trade off those things.

**Comment by hrz2000** [comment]
tks for your quick reply, got it.
